.devpanel/init.sh

```diff
@@ -15,31 +15,16 @@
 # For GNU Affero General Public License see <https://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------
 
-STATIC_FILES_PATH="$WEB_ROOT/sites/default/files"
-SETTINGS_FILES_PATH="$WEB_ROOT/sites/default/settings.php"
 #== Clone source code
 if [ ! -d "$APP_ROOT/starshot-prototype" ]; then
-  git clone https://github.com/phenaproxima/starshot-prototype.git starshot-prototype
+  git clone -b install-fails-with-database-url-argument https://github.com/darrenoh/starshot-prototype.git
```

Comment:
You should not use your own repository.

Reply:
This is a temporary fix to allow the database settings to be passed to the Starshot install scripts as an argument. With the official repository this would cause the install scripts to fail. I have a pull request to fix this. Once it is merged, we can use the official repository.
phenaproxima/starshot-prototype#127